### DISCUSSION TITLE
Styling not loading correctly on build but correctly on Dev

### DISCUSSION THREAD

**Discussion by aabdelw1** [gatsbyjs/gatsby#31759] [Help]
## Description

So I am in the finally stages of my project and as I am deploying on Netlify, I find an issue that didn't occur to me before where all my styles get completely wiped out on first reload. After you navigate to another page it disappears but if you do a hard refresh (command + shift + r) it reappears. I have scowered the docs, trying every possible solution including adding those additional plugins such as "gatsby-plugin-styled-components" but still nothing. The link to my website is below along with its repo:

### Steps to reproduce

Here is my website: https://ammarnasr.dev/

When you navigate to it, run a hard refresh (command + shift + r) and you should see text overlapping an image: 

If you'd like to replicate , here is my github repo: https://github.com/aabdelw1/personal
It works fine on develop, so run a "gatsby build" then "gatsby serve" to see if locally

### Expected result

<img width="1114" alt="Screen Shot 2021-06-02 at 10 22 46 PM" src="https://user-images.githubusercontent.com/19593635/120581809-11512f80-c3f1-11eb-86c6-cd3482909ef1.png">

### Actual result

<img width="1205" alt="Screen Shot 2021-06-02 at 10 22 04 PM" src="https://user-images.githubusercontent.com/19593635/120581753-f7175180-c3f0-11eb-83da-e2836bae942a.png">

### Environment

` System:
    OS: macOS 11.3.1
    CPU: (8) x64 Intel(R) Core(TM) i7-6820HQ CPU @ 2.70GHz
    Shell: 5.8 - /bin/zsh
  Binaries:
    Node: 12.16.1 - /usr/local/bin/node
    Yarn: 1.3.2 - /usr/local/bin/yarn
    npm: 6.13.4 - /usr/local/bin/npm
  Languages:
    Python: 3.8.3 - /usr/local/anaconda3/bin/python
  Browsers:
    Chrome: 91.0.4472.77
    Firefox: 89.0
    Safari: 14.1
  npmPackages:
    gatsby: ^2.31.1 => 2.32.13
    gatsby-cli: ^2.18.0 => 2.18.0
    gatsby-image: ^2.2.34 => 2.8.0
    gatsby-plugin-google-analytics: ^2.1.35 => 2.8.0
    gatsby-plugin-manifest: ^2.2.31 => 2.9.1
    gatsby-plugin-netlify: ^2.1.30 => 2.8.0
    gatsby-plugin-offline: ^3.0.27 => 3.7.1
    gatsby-plugin-postcss: ^2.3.6 => 2.3.13
    gatsby-plugin-react-helmet: ^3.1.16 => 3.7.0
    gatsby-plugin-react-svg: ^2.1.2 => 2.1.2
    gatsby-plugin-sass: ^2.1.26 => 2.8.0
    gatsby-plugin-scroll-reveal: 0.0.7 => 0.0.7
    gatsby-plugin-sharp: ^2.3.5 => 2.11.2
    gatsby-plugin-styled-components: ^3.10.0 => 3.10.0
    gatsby-source-filesystem: ^2.1.40 => 2.8.1
    gatsby-transformer-sharp: ^2.3.7 => 2.9.0
  npmGlobalPackages:
    gatsby-cli: 3.4.1`

**Comment by KyleAMathews** [comment]
Could you try enabling the `DEV_SSR` flag? That should help you debug the issue https://github.com/gatsbyjs/gatsby/discussions/28138

**Comment by aabdelw1** [comment]
I attempted this and still got nothing.
